auto_inertial_params/proposal.md

```diff
@@ -0,0 +1,414 @@
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>
+
+# Automatically Compute Inertia Tensor Proposal
```

Comment:
Any reason why this title is different from the title in manifest.xml?

Reply:
Ohh thank you for pointing that out. I have corrected the titles in both in [8eba84c](https://github.com/gazebosim/sdf_tutorials/pull/92/commits/8eba84c738ecc39bebe599aba4e9adf5d5e599cb)